protocol/message_handler.go

```diff
@@ -233,6 +233,9 @@ func (m *MessageHandler) handleCommandMessage(state *ReceivedMessageState, messa
 	// Increase unviewed count
 	if !common.IsPubKeyEqual(message.SigPubKey, &m.identity.PublicKey) {
 		chat.UnviewedMessagesCount++
+		if message.Mentioned {
```

Comment:
Maybe we can extract similar pieces of work with unviewed messages and notifications from `handleCommandMessage` and `HandleChatMessage`?